homeassistant/components/sabnzbd/sensor.py

```diff
@@ -78,21 +78,18 @@ class SabnzbdSensorEntityDescription(SensorEntityDescription, SabnzbdRequiredKey
         key="day_size",
         name="Daily Total",
         native_unit_of_measurement=DATA_GIGABYTES,
-        entity_registry_enabled_default=False,
```

Comment:
I don't think we should change this. They were disabled by default for a reason.

https://github.com/home-assistant/core/pull/68138#discussion_r855759878

Reply:
I was looking for the reason I did it in the first place and couldn't find it. So we'll leave like this by design. I'll add to the documentation that it's disabled by default and close this PR.
Thanks